packages/runtime-common/helpers/ai.ts

```diff
@@ -409,21 +426,25 @@ export function getSearchTool() {
           filter: {
             type: 'object',
             properties: {
-              type: {
-                type: 'object',
-                properties: {
-                  module: {
-                    type: 'string',
-                    description: `the absolute path of the module`,
-                  },
-                  name: {
-                    type: 'string',
-                    description: 'the name of the module',
-                  },
+              every: {
+                type: 'array',
+                items: {
+                  anyOf: [
+                    {
+                      type: 'object',
+                      properties: { type: cardTypeFilterProperty },
+                      required: ['type'],
+                    },
+                    {
+                      type: 'object',
+                      properties: { contains: containsFilterProperty },
+                      required: ['contains'],
+                    },
+                  ],
```

Comment:
does `every` mean both cardType and contains filters need to exist in the query? I think it's sufficient to just have the contains, since we probably want all cards that contain the string in their title no matter their type.

Reply:
Yea  agreed `every` is not needed I think. I made a comment about it